srcpkgs/python-soupsieve/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'python-soupsieve'
+pkgname=python-soupsieve
+version=1.7.3
+revision=1
+archs=noarch
+wrksrc="soupsieve-${version}"
+build_style=python-module
+pycompile_module="soupsieve"
+hostmakedepends="python-setuptools python3-setuptools"
+checkdepends="python-pytest python-lxml python-html5lib python-BeautifulSoup4
+ python-backports.functools_lru_cache python3-pytest python3-lxml
+ python3-html5lib python3-BeautifulSoup4"
+short_desc="CSS4 selector implementation for Python2 Beautiful Soup"
+maintainer="Aluísio Augusto Silva Gonçalves <aluisio@aasg.name>"
+license="MIT"
+homepage="https://facelessuser.github.io/soupsieve/"
+distfiles="https://github.com/facelessuser/soupsieve/archive/${version}.tar.gz"
```

Comment:
Is it better to fetch from `${PYPI_SITE}` like other python module.
`${PYPI_SITE` also provide their own sha256 hash.
Something like:

```
distfiles="${PYPI_SITE}/s/soupsieve/soupsieve-${version}.tar.gz"
checksum=87db12ae79194f0ff9808d2b1641c4f031ae39ffa3cab6b907ea7c1e5e5ed445
```